server/src/DBService/Exposure/Query.ts

```diff
@@ -44,25 +44,38 @@ query ExposureByInvestigationId ($investigationId: Int!){
 `;
 
 export const GET_EXPOSURE_SOURCE_OPTIONS = gql`
-query getOptionalExposureSources($searchRegex: String!) {
-    allCovidPatients(filter: {or: [{fullName: {like: $searchRegex}}, {identityNumber: {like: $searchRegex}}, {primaryPhone: {like: $searchRegex}}]}) {
-      nodes {
-        fullName
-        identityNumber
-        primaryPhone
-        epidemiologyNumber
-        birthDate
-        addressByAddress {
-            cityByCity {
-              displayName
+    query getOptionalExposureSources($searchValue: String! , $searchInt: Int! , $searchStartDate : Datetime! , $searchEndDate : Datetime!) {
+        allCovidPatients(
+            filter: {
+                or: [
```

Comment:
why not sending filterBy from the client as we did in usersManagement and investigationTable

Reply:
🤷‍♂️ . but when I think about it it's more safe like that , as a user with filterBy I can send to the endpoint any parameters I want inside the filterBy (not just the fields I need to search by) where here I'm leaving the logic to the server side.